docs/en/01_Working_with_projects/03_Recipes_and_supported_modules.md

```diff
@@ -81,23 +80,9 @@ See [upgrading guide](upgrading) for instructions.
 ## Recipe contents
 
 To obtain the composition of the latest stable recipe, the easiest way is to head to the
-[repository](https://gitlab.cwp.govt.nz/cwp/cwp-recipe-basic/) and look at the composer file and the dependencies
+[recipe repositories](https://github.com/topics/cwp) and look at the respective composer file and the dependencies
```

Comment:
I think this is talking about the contents of each recipe. Perhaps instead we could say something like "To obtain the composition of the latest stable recipe, the easiest way is to head to the recipe's repository and look at the respective composer file, then update the dependencies contained there. For example, [here is the blog recipe contents](https://github.com/silverstripe/recipe-blog/blob/master/composer.json)."